samaritan/templates/samaritan/views/group_members_view.html

```diff
@@ -20,6 +20,11 @@ <h3>Group: <i id="group-name"></i></h3>
       <button type="button" class="btn btn-info btn-md" id="email-group-members">
         <i class="fa fa-envelope-o fa-fw"></i> Email
       </button>
+      
+      <!-- Button sms modal-->
+      <button type="button" class="btn btn-success btn-md" id="sms-group-members">
+        <i class="fa fa-comment fw" aria-hidden="true"></i> Message
```

Comment:
sorry I missed that. It needs to be fa-fw not fw

Reply:
done